gyronimo/metrics/morphism_helena.cc

```diff
@@ -17,67 +17,111 @@
 
 // @morphism_helena.cc, this file is part of ::gyronimo::
 
-#include <numbers>
-#include <gyronimo/core/error.hh>
-#include <gyronimo/core/multiroot.hh>
 #include <gyronimo/metrics/morphism_helena.hh>
+#include <gsl/gsl_vector.h>
+#include <gsl/gsl_multiroots.h>
```

Comment:
Low-level gsl operations should be handled by `multiroot` alone. Are these gsl includes really needed here?